packages/wrangler/src/dev/local.tsx

```diff
@@ -251,12 +251,16 @@ function useLocalWorker({
         process.stdout.write(data);
       });
 
+      // parse the node inspector url (which may be received in chunks) from stderr
+      let stderrData = "";
       local.current.stderr?.on("data", (data: Buffer) => {
+        stderrData += data.toString();
         const matches =
-          /Debugger listening on (ws:\/\/127\.0\.0\.1:\d+\/[A-Za-z0-9-]+)/.exec(
-            data.toString()
+          /Debugger listening on (ws:\/\/127\.0\.0\.1:\d+\/[A-Za-z0-9-]{36})/.exec(
```

Comment:
Is the ID always 36 chars long? I understand you are doing this to prevent picking up only some of the ID if it breaks across two chunks, but I wonder if we could be more lenient by looking for a terminating character, such as `\n` or whitespace or something?

Reply:
The ID portion of the URL currently returned by `node --inspect` appears to be a UUID which is always 36 characters long, but I've updated the regex to capture up to `\r` or `\n` as per your suggestion - does that work?

Reply:
Alternatively, the regex could be loosened up further to something like `/Debugger listening on (ws:\/\/.+:\d+\/.+)[\r|\n]/`  which was a slight adaption of https://github.com/nodejs/node/blob/528fef2285b48d66f2ec7b41696da4b596c87c31/test/common/inspector-helper.js#L375